src/parser/rogue/outlaw/CombatLogParser.js

```diff
@@ -39,6 +40,7 @@ class CombatLogParser extends CoreCombatLogParser {
     restlessBlades: RestlessBlades,
     rolltheBonesBuffs: RollTheBonesBuffs,
     //Items
+    deadshot: DeadShot,
```

Comment:
While technically correct, I'd suggest naming the imported module "Deadshot" instead.

Reply:
Corrected.  Now reads:

```deadshot: Deadshot,```